bindings/py/src/nupic/bindings/tools/serialization_test_py_region.py

```diff
@@ -69,41 +58,8 @@ def randomSeed(self):
     return self._rand.getSeed()
 
 
-  @staticmethod
-  def getSchema():
-    """Return the pycapnp proto type that the class uses for serialization.
-
-    This is used to convert the proto into the proper type before passing it
-    into the read or write method of the subclass.
-    """
-    return SerializationTestPyRegionProto
-
-
-  def writeToProto(self, proto):
-    """Write state to proto object.
-
-    The type of proto is determined by getSchema().
-    """
-    proto.dataWidth = self._dataWidth
-    self._rand.write(proto.random)
-
 
   @classmethod
```

Comment:
remove this line

Reply:
No, executeMethod() that follows needs to be declared as a classmethod.  Otherwise I think it would be declared abstract.

Reply:
Do you reply to a good line/file? There's no executeMethod here. The decorator belonged to 'readFromProto', and now would fall down to 'initialize'

Reply:
Hmmm, I must have been looking at the wrong file.  Removed line 62 of serialization_test_py_region.py